style.css

```diff
@@ -1,2 +1,215 @@
+  padding-right: 63px;
+}
+
+.nav__link:nth-child(2) {
+  /*padding-left: 63px;*/
+  padding-right: 25px;
+}
+
+.nav__link:nth-child(3) {
+  padding-right: 44px;
+}
+
+.nav__link:nth-child(4) {
+  padding-right: 46px;
+}
+
+.nav__link:nth-child(5) {
+  padding-right: 221px;
+}
+
+.nav__link:nth-child(6) {
+  padding-right: 30px;
+}
+
+.nav:nth-child(7) {
+  padding-right: 30px;
+}
+
+.nav__link:hover {
+  color: #f00;
+  text-decoration: underline;
+}
+
+.intro {
+  background-image: url(images/sectionone/wordmark.png);
+  margin-left: -74px;
+  margin-top: 75px;
+  height: 467px;
+  width: 1514px;
+}
+
+.down {
+  height: 14px;
+  margin-top: 121px;
+  width: 28px;
+}
+
+.intro__title {
+  color: #000;
+  font-size: 90px;
+  font-weight: 600;
+  margin-bottom: 8px;
+  padding-left: 609px;
+  padding-right: 535px;
+  padding-top: 1px;
+}
+
+.intro__title-small {
+  color: #000;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 18px;
+  margin-bottom: 0;
+  margin-top: 1px;
+  padding-bottom: 26px;
+  padding-left: 674px;
+  padding-right: 579px;
+}
+
+.intro__car {
+  height: 450px;
+  padding-right: 273px;
+  margin-left: 273px;
+  margin-right: 273px;
+  width: 968px;
+}
+
+.features {
+  display: flex;
+  margin-left: 321px;
+  margin-right: 321px;
+  /* margin-top: 154px; */
+}
+
+.time__speedometer {
+  /* margin-bottom: 1899px; */
+  /* margin-left: 321px; */
+  /* margin-right: 14px; */
+  margin-top: 166px;
+  padding-right: 14px;
+  /* box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25); */
+  /* transform: matrix(-1, 0, 0, 1, 0, 0); */
+}
+
+.time__two-point-seven {
+  color: #000;
+  font-weight: 600;
+  font-size: 35px;
+  line-height: 43px;
+  /* margin-right: 1008px; */
+  margin-top: 154px;
+}
+
+.time__sec {
+  color: #000;
+  font-weight: 600;
+  font-size: 20px;
+  left: 438px;
+  line-height: 24px;
+  top: 770px;
+}
+
+.time__speed {
+  color: #000;
+  font-weight: normal;
+  font-size: 12px;
+  left: 381px;
+  line-height: 15px;
+  top: 796px;
+}
+
+.distance__value {
+  color: #000;
+  font-size: 35px;
+  font-weight: 600;
+  left: 597.5px;
+  line-height: 43px;
+  top: 754px;
+}
+
+.distance__km {
+  color: #000;
+  font-size: 20px;
+  font-weight: 600;
+  left: 667px;
+  line-height: 24px;
+  top: 769px;
+}
+
+.distance__resource {
+  color: #000;
+  font-size: 12px;
+  font-weight: normal;
+  left: 597.5px;
+  line-height: 15px;
+  top: 795px;
+}
+
+.test__drive {
+  color: #000;
+  font-size: 13px;
+  font-weight: bold;
+  left: 830px;
+  line-height: 16px;
+  top: 777px;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.order-first {
+  background-color: #fff;
+  border: 2px solid #e51937;
+  box-sizing: border-box;
+  color: #e51937;
+  font-size: 13px;
+  font-weight: bold;
+  height: 40px;
+  left: 966px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  top: 776px;
+  width: 160px;
+}
+
+.interior__pic {
+  height: 844px;
+  margin-top: 248px;
+  width: 1548px;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/IFW/pullRequest?prid=6044074)